src/System.Private.ServiceModel/tools/test/Bridge/Bridge/Program.cs

```diff
@@ -21,6 +21,8 @@ internal class Program
 
         private static void Main(string[] args)
         {
+            AppDomain.CurrentDomain.UnhandledException +=
+                new UnhandledExceptionEventHandler(BridgeUnhandledExceptionHandler);
```

Comment:
It's possible we need to do this in another place -- AppDomainManager.  Bridge.exe runs in one AD, and resources are hosted in a 2nd AD, created by AppDomainManager.  The crash we saw happened in the 2nd AD.  So though it did not occur in the Bridge's "primary" AD, it did take down the Bridge.exe process.

LGTM -- subject to what you think about another handler for the 2nd AD.